class_item.rb

```diff
@@ -0,0 +1,32 @@
+require 'securerandom'
+
+# class_item.rb
+class Item
```

Comment:
- [ ] Please make you write a unit test for your item class and ensure to add a test for the methods `can_be_archived?()` and `move_to_archived()` in the item class. :pray:

Reply:
thank you @Owusu-Desmond 
We add a new test file: class_item_spec.rb with the tests required. ✔️👍🏽